### PR TITLE
fix(content): Fix pronouns in first contact missions

### PR DESCRIPTION
**Pull request by tibetiroka**
**Fix:** Fixes pronouns in the first contact missions of the Bunrodea, Coalition and the Wanderers by changing `it` to `they` after communication is established. This avoids dehumanizing these alien species.

## Fix Details

Most of the other species were unaffected. The Korath first contact also uses `it`, but that's being reworked by Unordered in the Korath campaign PR.

~~The Quarg missions also use `it`, but that seems to be the norm for every mission, not just the first contact. I'm waiting for more input on what to do with that species.~~ It seems the Quarg are indeed `it`.

For most species, other missions already use `they` or, more commonly, gendered pronouns. I'd also support switching the `they` to gendered pronouns here, but I leave it up to the faction authors to figure out who has what gender, or if they want it to be specified at all.
## Testing Done

Shouldn't cause issues.

### CODE REVIEW

**Review by lumbar527 — APPROVED** [review]
I think this is good, follows up your #8775 PR.

File: data/bunrodea/bunrodea missions.txt [hunk 1]
```diff
@@ -205,15 +205,15 @@ mission "First Contact: Bunrodea (Hostile)"
...
 				goto end
 			
 			label slighted
-			`	The Queen motions to one of the guards and says something to it that the creature does not translate. She then speaks to you, and the creature translates: "Since it is my guards that have shown you disrespect, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
+			`	The Queen motions to one of the guards and says something to them that the creature does not translate. She then speaks to you, and the creature translates: "Since it is my guards that have shown you disrespect, you will be allowed to live. My guards have been ordered to leave you be, and you may come and go on a small number of our worlds. But this is only so that you do not cause further trouble, and to allow you to refuel if you are passing through my systems. Your presence will only be tolerated, not welcomed. Now please leave my court."`
```
Comment (MasterOfGrey) [inline data/bunrodea/bunrodea missions.txt:216]: Technically the guards could justifiably be "it" - we have established that these are insectoids with a queen, I don't think know if the guards actually have personhood.

Comment (tibetiroka) [reply data/bunrodea/bunrodea missions.txt:216]: I'd prefer `they`, but let's ask @Amazinite.

**Comment by W1zrad** [comment]
Changes look good. As Grey noted, I do think it's worth asking @Amazinite about whether Bunrodea should be `it` or `they`.

File: data/bunrodea/bunrodea missions.txt (hunk 1 above)
Comment (Ferociousfeind) [reply data/bunrodea/bunrodea missions.txt:216]: I think humans would implicitly assume all the individual creatures are bestowed with personhood- it's a cultural difference you can confront later.

Comment (MasterOfGrey) [reply data/bunrodea/bunrodea missions.txt:216]: I personally would not assume that the unspeaking bug warriors - who only "speak" robotically with cybernetic implants in a specific entry point to the mission, of what appears to be an insectoid caste-based alien race with a Queen - were in possession of personhood.

File: data/bunrodea/bunrodea missions.txt [hunk 2]
```diff
@@ -114,7 +114,7 @@ mission "First Contact: Bunrodea (Hostile)"
 			choice
 				`	"I'm not the first human here?"`
 				`	"What do you mean one hundred years ago?"`
-			`	"Human visitors, many years ago," it says. "Not important. The Queen desires yours presence." It then says something in a language that the mantis-like aliens understand, and they motion for you to follow as the small alien begins making its way back to the steps of the palace. You have a feeling you're going to dread the next half hour.`
+			`	"Human visitors, many years ago," they say. "Not important. The Queen desires yours presence." They then say something in a language that the mantis-like aliens understand, and they motion for you to follow as the small alien begins making their way back to the steps of the palace. You have a feeling you're going to dread the next half hour.`
```
Comment (Amazinite) [inline data/bunrodea/bunrodea missions.txt:117]: Revert this line and keep the creature as an "it".

**Review by W1zrad — APPROVED** [review]

**Pull request merged**